tools/osbuild-mpp

```diff
@@ -491,6 +559,19 @@ class DepSolver:
         return deps
 
 
+class DepSolverFactory():
+
+    def __init__(self, cachedir, persistdir):
+        self._cachedir = cachedir or "/tmp/pacsolve"
```

Comment:
maybe something more generic than `pacsolve`? `/tmp` being a `tmpfs` it won't survive a reboot, maybe that is ok. This will change the behaviour of the dnf depsolver, since `None` for `cachedir` will lead to it using the global dnf one. so Maybe just remove the `or /tmp/...` here. It is still present in the `PacmanSolver` constructor so nothing should change.

Reply:
Okido, dropped it for the original DepSolver.